1.MLP_Binary_Classifier/MLP_utils/utils.py

```diff
@@ -0,0 +1,881 @@
+"""
+Functions for Machine Learning Model optimization, training and testing.
+These are helper functions meant to be called in a separate notebook or script
+"""
+
+
+import ast
+from configparser import ConfigParser
+from typing import Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+import optuna
+import pandas as pd
+import seaborn as sns
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from MLP_utils.parameters import Parameters
+from sklearn.metrics import (
+    accuracy_score,
+    auc,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+    precision_score,
+    recall_score,
+    roc_auc_score,
+    roc_curve,
+)
+from sklearn.model_selection import train_test_split
+
+config = ConfigParser()
+config.optionxform = str
+config.read("MLP_utils/config.ini")
+
+config["DEFAULT"]
```

Comment:
I think this is a debug line maybe? Not sure if it is doing anything, consider deleting